Well-Known-Specification.md

```diff
@@ -0,0 +1,68 @@
+# The "related-website-set.json" Well-Known Resource Identifier
+
+This page describes the use and content of URIs having the path 
+`/.well-known/related-website-set.json`.
+## Use
+
+A page may be hosted at the URI having the path of 
+`/.well-known/related-website-set.json` in order to demonstrate
+administrative access to the site when submitted as an entry in a Related 
+Website Site (RWS).  
```

Comment:
Please use active voice to make this clearer:

```suggestion
A site may demonstrate intentional opt-in to participating in a Related Website Set (RWS) by hosting a specific JSON file at the URI with a path of
`/.well-known/related-website-set.json`.
```

Reply:
Done